digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -920,6 +920,27 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
     }
   }
 
+  private val getReservedStatusOfJunctionPoints: SwaggerSupportSyntax.OperationBuilder = (
+    apiOperation[Map[String, Any]]("getReservedStatusOfJunctionPoints")
+      .parameters(
+        queryParam[Long]("ids").description("Junction point id:s")
+      )
+      tags "ViiteAPI - NodesAndJunctions"
+      summary "Checks if the road that the junction points are on, is reserved to a road address project"
```

Comment:
Checks if the junction points are on a road part that is reserved to a road address project.
tai
Checks whether any of the given junction points belongs to a road part currently reserved to a road address project.

(Pääpointti: tsekataan junction pointien tilaa, ei teiden, vaikka se toki tievarauksen kautta selvitetäänkin)